raster.py

```diff
@@ -459,23 +459,28 @@ def d13C_var_geotiff() -> AmazonGeoTiff:
   "ordinary_kriging_linear_d18O_predicted_variance" : krig_variances_isoscape_geotiff
 }
 
+# Converts a resolution into a Bounds class, storing info about pixel size and raster size.
+def create_bounds_from_res(res_x: int, res_y: int, brazil_bounds: Bounds):
+  # Use brazil_bounds to get the min/max lat and lon. Scale everything else
```

Comment:
Do we want to change this from brazil_bounds to input_bounds instead?

Reply:
Yes